static/customWidgets/GeometryWidget/geometrywidget.js

```diff
@@ -34,33 +34,22 @@
       }),
     });
 
-    ;
-
-    var layers =  this.options.wms_layers.split(',');
-    var ol_layers = [];
-    for (var i=0; i < layers.length; i++) {
-        ol_layers.push(
-            new ol.layer.Image({
-            source: new ol.source.ImageWMS({
-              url: layers[i],
-              projection: "EPSG:2056",
-            }),
-            opacity: 1,
-          })
+    var layer_list = [];
+
+    this.options.wms_layers.split(",").forEach(function (item, index) {
+      layer_list.push(
+        new ol.layer.Image({
+          source: new ol.source.ImageWMS({
+            url: item,
+            projection: "EPSG:2056",
```

Comment:
Use CRS parameter from WMS here?

Reply:
=> The whole widget is for EPSG:2056

Reply:
Then what if people want to upload WGS84 KML files? Or WorldMercator GPX data? Maybe we should consider that in the future as well?

Reply:
=> KML are reprojected on import => https://github.com/yverdon/geocity/blob/master/static/customWidgets/GeometryWidget/geometrywidget.js#L308